features/support/env.rb

```diff
@@ -28,3 +29,14 @@
 Capybara.javascript_driver = :chrome
 
 World(FactoryBot::Syntax::Methods)
+Cucumber::Rails::Databse.javascript_strategy = :truncation
+WebMock.disable_net_connect!(allow_localhost: true)
+Before do
+  stub_request(:get, https://newsapi.org/v2/everything?q=bitcoin&apiKey=62aeaa7f151d4e21a0d8bcc6032044ea ).
```

Comment:
This needs to be a string @Noelryn

Reply:
Got it, I am currently in the process for fixing this